Chutzpah/TestRunner.cs

```diff
@@ -288,7 +288,7 @@ private IChutzpahWebServerHost SetupWebServerHost(ConcurrentBag<TestContext> tes
             var needsServer = options.Engine.GetValueOrDefault() != Engine.Phantom;
 
             IChutzpahWebServerHost webServerHost = null;
-            var contextUsingWebServer = testContexts.Where(x => x.TestFileSettings.Server?.Enabled.GetValueOrDefault() == true).ToList();
+            var contextUsingWebServer = testContexts.Where(x => x.TestFileSettings.Server.Enabled.HasValue).ToList();

```

Comment:
Couple things here: 
1. I was guarding against Server being null, which you removed. Do you see that is not needed?
2. Why changing to HasValue versus getting the actual value? We just want contexts that have it enabled right?

Reply:
1. Good point, Was testing against Chrome and now going back tested phantom again and looks like it could be null.

2. I think there was a bug. 
First, when engine is chrome,  var needsServer is true.
Second when Server.Enabled is false, it was treated same as absent, so previous statement does not select it, and contextUsingWebServer and contextWithChosenServerConfiguration are both null.
So, the next statement kicked in (if (contextWithChosenServerConfiguration == null && needsServer), and forced a server instance to it anyway, effectively ignoring "server:false" setting.